src/documents/SubcomponentDeclaration.md

```diff
@@ -91,39 +96,48 @@ The properties allowed at top level in the subcomponent record are as follows:
 }</code></pre></td>
         </tr>
         <tr>
-            <td><code>createOnEvent</code> (Optional)</td>
-            <td>String</td>
-            <td><p>Specifies an event that will trigger the creation of the subcomponent. This option is used when the subcomponent should not be created immediately as part of the construction process of its parent, but rather at a later time signalled by the firing of the specified event. If this value is a simple string, it represents an event held on the parent component - it may also take the form of an <a href="IoCReferences.md">IoC reference</a> to an event elsewhere in the component tree.</p>
-
-<p>Note that if the specified event fires multiple times, the corresponding component will be destroyed and then recreated on every firing of the event after the first time.</p></td>
+            <td><code>createOnEvent</code> [optional]</td>
+            <td><code>String</code></td>
+            <td><p>Specifies an event that will trigger the creation of the subcomponent. 
+            This option is used when the subcomponent should not be created immediately as part of the construction process of its parent, but rather at a later time signalled by the firing of the specified event. 
+            If this value is a simple string, it represents an event held on the parent component - it may also take the form of an <a href="IoCReferences.md">IoC reference</a> to an event elsewhere in the component tree.</p>
+            <p>Note that if the specified event fires multiple times, the corresponding component will be destroyed and then recreated on every firing of the event after the first time.</p></td>
```

Comment:
Perhaps we should change this and the other notes in these files to use the `infusion-docs-note` styling.